README.md

```diff
@@ -47,6 +47,16 @@ $ cargo build --release
 
 This will create the `lucky_commit` binary (`lucky_commit.exe` on Windows) in the `target/release` directory. You can move this to wherever you want, or set up an alias for it.
 
+#### Distro packages
+
+##### Arch Linux
+
+`lucky-commit` can be installed from the [community repository](https://archlinux.org/packages/community/x86_64/lucky-commit/) using [pacman](https://wiki.archlinux.org/title/Pacman):
+
+```
+$ pacman -S lucky-commit
```

Comment:
Just to clarify, if an Arch Linux user follows these instructions, do they still need to have installed Rust/Cargo previously? Or will those get installed automatically as part of installing this package?

Reply:
No, they don't need to install Rust previously. Running the `pacman` command will pull the pre-built binary package from Arch repositories and install it to the correct location. Rust is only needed if you want to build the package yourself.

```
[root@orhun ~]# pacman -S lucky-commit
resolving dependencies...
looking for conflicting packages...

Packages (1) lucky-commit-2.0.5-1

Total Download Size:   0.16 MiB
Total Installed Size:  0.39 MiB

:: Proceed with installation? [Y/n] 
:: Retrieving packages...
 lucky-commit-2.0.5-1-x86_64                         163.4 KiB   263 KiB/s 00:01 [##############################################] 100%
(1/1) checking keys in keyring                                                   [##############################################] 100%
(1/1) checking package integrity                                                 [##############################################] 100%
(1/1) loading package files                                                      [##############################################] 100%
(1/1) checking for file conflicts                                                [##############################################] 100%
(1/1) checking available disk space                                              [##############################################] 100%
:: Processing package changes...
(1/1) installing lucky-commit                                                    [##############################################] 100%
```

You can take a look at the build instructions [here](https://github.com/archlinux/svntogit-community/blob/packages/lucky-commit/trunk/PKGBUILD).